src/interpreter.h

```diff
@@ -92,73 +92,71 @@ static const IstreamOffset kInvalidIstreamOffset = ~0;
 #define WABT_TABLE_ENTRY_DROP_OFFSET sizeof(uint32_t)
 #define WABT_TABLE_ENTRY_KEEP_OFFSET (sizeof(IstreamOffset) + sizeof(uint32_t))
 
-enum class InterpreterOpcode {
+enum class Opcode {
```

Comment:
Won't this make using simple tools like grep to find the definition of OpCode a little harder?  I guess it fine..